.eslintrc.yaml

```
@@ -192,6 +193,7 @@ rules:
   jquery/no-val: [0]
   jquery/no-when: [2]
   jquery/no-wrap: [2]
+  no-jquery/no-event-shorthand: [3.5]
```

```suggestion
  no-jquery/no-event-shorthand: [2]
```

0 = disabled
1 = warning
2 = error

I assume this will raise a bunch of errors after this change.